docs/howto/use_modules_and_layers.md

```diff
@@ -313,24 +323,24 @@ val fullRepo: Layer[Nothing, UserRepo] = connectionLayer >>> postgresLayer
 ```
 
 ## Layers are shared in the dependency graph
-One important feature of `ZIO` layers is that they are acquired in parallel wherever possible, and they are shared. For every layer in our dependency graph, there is only one instance of it that is shared between all the layers that depend on it. If you don't want to share a module, create a fresh, non-shared version of it through `ZLayer.fresh`.
+One important feature of `ZIO` layers is that they are acquired in parallel wherever possible, and they are shared. For every layer in our dependency graph, there is only one instance of it that is shared between all the layers that depend on it. If you don't want to share a module, create a fresh, non-shared version of it through [`ZLayer.fresh`][ZLayer.fresh].
```

Comment:
Normally we would use `#` for methods on a trait and `.` for methods on the companion object.

Reply:
Just so I'm clear, what is the semantic distinction? Not disagreeing, curious. Is it about making it clear that one thing is a method call on a concrete implementation, and another is a call on something that may not be concrete? (In this case ZLayer is a sealed abstract class, not a trait, but I assume this is what you mean?)

Reply:
It is more that there are cases where the same method name is used on both a type and its companion object. For example, there is a `unit` method on the `ZIO` trait that transforms the result type to `Unit` so you could do `promise.succeed(()).unit` (ignoring the `Boolean` value returned by succeed indicating whether the promise had already been completed). There is also a `unit` method on the `ZIO` companion object that just constructs an effect that succeeds with the unit value. So it is useful to be able to distinguish between the two when talking about them.